hw/ip/lc_ctrl/lc_ctrl.core

```diff
@@ -19,7 +19,7 @@ filesets:
       - lowrisc:ip:otp_ctrl_pkg
       - lowrisc:ip:kmac_pkg
       - lowrisc:ip:rv_dm
-      - lowrisc:ip:alert_handler_reg
+      - lowrisc:ip_interfaces:alert_handler_reg
```

Comment:
hmm... i think i'm not completely following why it's called `opentitan` for rv_plic and `ip_interfaces` here